### PR TITLE
#5-Added carousel and Latest Events Section

### PR DESCRIPTION
**Pull request by himanshu-dhiman**

### CODE REVIEW

File: public/wp-content/themes/ColoredCow/landing.php [hunk 1]
```diff
@@ -13,10 +13,69 @@
...
+			<div class="col-sm-12 col-lg-6 col-md-6 latest-events">
+				<span class="theme">Theme</span><br> <br>
+		        <span class="icon"><i class='fa fa-calendar'  aria-hidden='true'></i>&nbsp;Date</span><br><br> 
+		        <span class="icon"><i class='fa fa-map-marker fa-lg' aria-hidden='true'></i>&nbsp;Venue</span>
+			</div>
+		</div>
+		<hr>
+		<div>
+			<h1 class="carousel-head"><i class="fa fa-camera-retro fa-1x"></i>&nbsp;Event Gallery</h1>
+			<div id="carouselExampleIndicators" class="carousel slide"  data-ride="carousel">
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php]: naming convention

File: public/wp-content/themes/ColoredCow/landing.php [hunk 2]
```diff
@@ -13,10 +13,69 @@
...
+					<li data-target="#carouselExampleIndicators" data-slide-to="1">	</li>
+					<li data-target="#carouselExampleIndicators" data-slide-to="2">	</li>
+					<li data-target="#carouselExampleIndicators" data-slide-to="3">	</li>
+					<li data-target="#carouselExampleIndicators" data-slide-to="4">	</li>
+				</ol>
+				<div class="carousel-inner" role="listbox">
+					<div class="carousel-item active">
+					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree1.jpg");?>" alt="First slide">
+					  	<div class="carousel-caption d-none d-md-block">
+							<h3>#SOIREE 1.</h3>
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php]: use css for font-size

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 3]
```diff
@@ -73,4 +73,31 @@ h5{
     background-color: $black;
     color: $white;
 }
+.theme{
+	@extend .marcellus-sc;
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: name font classes as: .font-font-name

**Review by Shubham2301 — CHANGES_REQUESTED** [review]

File: public/wp-content/themes/ColoredCow/landing.php [hunk 4]
```diff
@@ -23,43 +23,37 @@
...
+					<li data-target="#carousel-example-indicators" data-slide-to="1"></li>
+					<li data-target="#carousel-example-indicators" data-slide-to="2"></li>
+					<li data-target="#carousel-example-indicators" data-slide-to="3"></li>
 				</ol>
 				<div class="carousel-inner" role="listbox">
 					<div class="carousel-item active">
 					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree1.jpg");?>" alt="First slide">
 					  	<div class="carousel-caption d-none d-md-block">
-							<p class="carousel-head">#SOIREE 1</p>  
+						<p class="carousel-head">#SOIREE 1</p>
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php]: indent

File: public/wp-content/themes/ColoredCow/landing.php [hunk 5]
```diff
@@ -23,43 +23,37 @@
...
 					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree2.jpg");?>" alt="Second slide">
 					  	<div class="carousel-caption d-none d-md-block">
-							<p class="carousel-head">#SOIREE 2</p>  
-					  	</div>
-					</div>
-					<div class="carousel-item">
-					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree3.jpg");?>" alt="Third slide">
-					  	<div class="carousel-caption d-none d-md-block">
-							<p class="carousel-head">#SOIREE 3</p>  
+						<p class="carousel-head">#SOIREE 2</p>
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php]: indent

File: public/wp-content/themes/ColoredCow/landing.php [hunk 6]
```diff
@@ -13,10 +13,63 @@
...
+					  	</div>
+					</div>
+					<div class="carousel-item">
+					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree2.jpg");?>" alt="Second slide">
+					  	<div class="carousel-caption d-none d-md-block">
+						<p class="carousel-head">#SOIREE 2</p>  
+					  	</div>
+					</div>
+					<div class="carousel-item">
+					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree4.jpg");?>" alt="third slide">
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: always prefer single quotes in PHP. Single and double quotes work differently in PHP.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 7]
```diff
@@ -13,10 +13,63 @@
...
+				</ol>
+				<div class="carousel-inner" role="listbox">
+					<div class="carousel-item active">
+					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree1.jpg");?>" alt="First slide">
+					  	<div class="carousel-caption d-none d-md-block">
+						<p class="carousel-head">#SOIREE 1</p>  
+					  	</div>
+					</div>
+					<div class="carousel-item">
+					  	<img class="d-block img-fluid images_carousel" src="<?php echo esc_url(get_template_directory_uri()."/rsvp_images/soiree2.jpg");?>" alt="Second slide">
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: prefer single quotes in PHP.

File: public/wp-content/themes/ColoredCow/landing.php [hunk 8]
```diff
@@ -13,10 +13,63 @@
...
+		        <span class="icon"><i class='fa fa-map-marker fa-lg' aria-hidden='true'></i>&nbsp;Venue</span>
+			</div>
+		</div>
+		<hr>
+		<div class="container carousel-section">
+			<div class="carousel-head"><i class="fa fa-camera-retro fa-1x"></i>&nbsp;Event Gallery</div>
+			<br>
+			<div id="carousel-example-indicators" class="carousel slide"  data-ride="carousel">
+				<ol class="carousel-indicators">
+					<li data-target="#carousel-example-indicators" data-slide-to="0" class="active"></li>
```
Comment (rathorevaibhav) [inline public/wp-content/themes/ColoredCow/landing.php]: carousel-example???? Can you name your carousel better?

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]
Just some feedback. Everything else looks good.

**Review by Shubham2301 — CHANGES_REQUESTED** [review]

**Pull request merged**